pkg/controller/networkpolicy/tier.go

```diff
@@ -158,3 +166,31 @@ func (n *NetworkPolicyController) initTier(t *secv1alpha1.Tier) {
 		return
 	}
 }
+
+// initTierUpdates attempts to update Tiers using an
+// exponential backoff period from 1 to max of 8secs.
+func (n *NetworkPolicyController) initTierUpdates(t *secv1alpha1.Tier) {
```

Comment:
strange name. why not just `updateTier`?

Reply:
sure.. i just added the init part because it is performed during the init stage.. I'm fine changing to the suggested name